src/libultra/io/motor.c

```diff
@@ -45,7 +45,7 @@ s32 osSetRumble(OSPfs* pfs, u32 vibrate) {
     return ret;
 }
 
-void osSetUpMempakWrite(s32 channel, OSPifRam* buf) {
+void _MakeMotorData(s32 channel, OSPifRam* buf) {
```

Comment:
I looked into this before for Majora's Mask libultra, this is the original name. It has such an unusual name because it was originally marked static, so wouldn't have been visible outside of this file. I confirmed this by looking in the mdebug section of a libultra.a that contained motor.o